session-state/background.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Add a context menu item to borderify the current tab.
+ */
+browser.menus.create({
+  id: "borderify",
```

Comment:
minor nit, but if someone had this and borderify installed this might be confusing